.github/workflows/cleanup-branch-builds.yml

```diff
@@ -28,7 +28,3 @@ jobs:
         # When set to 0, all deletable versions will be deleted.
         # When set greater than 0, all deletable package versions except the specified number will be deleted.
         min-versions-to-keep: 0
-
-        # Can be a single package version id, or a comma separated list of package version ids.
-        # Defaults to an empty string.
-        package-version-ids: "${{ steps.version.outputs.ids }}"
```

Comment:
Why are you removing this?

Reply:
if you look at the build action, it's already deleting the "ids". When it runs again, the array has the list of deleted ids and tries to delete them again :-> check https://github.com/liquibase/liquibase/actions/runs/5479562981/jobs/9981564213